src/test/kotlin/no/nav/emottak/Database.kt

```diff
@@ -3,28 +3,26 @@ package no.nav.emottak
 import app.cash.sqldelight.driver.jdbc.JdbcDriver
 import app.cash.sqldelight.driver.jdbc.asJdbcDriver
 import arrow.core.memoize
-import com.zaxxer.hikari.HikariConfig
 import io.kotest.core.extensions.install
 import io.kotest.core.spec.Spec
 import io.kotest.extensions.testcontainers.JdbcDatabaseContainerExtension
 import no.nav.emottak.queries.PayloadDatabase
 import org.flywaydb.core.Flyway
 import org.flywaydb.core.api.output.MigrateResult
 import org.testcontainers.containers.PostgreSQLContainer
-import javax.sql.DataSource
 
 private const val MIGRATIONS_PATH = "filesystem:./build/generated/migrations"
-private const val SMTP_TRANSPORT_DB_NAME = "emottak-smtp-transport-db"
+private const val TEST_DATA_PATH = "filesystem:./src/test/resources/testDb"
 
 fun Spec.payloadDatabase() = PayloadDatabase(jdbcDriver())
-fun DataSource.asPayloadDatabase() = PayloadDatabase(asJdbcDriver())
+fun stopContainer() = container().stop()
```

Comment:
Det er overflødig og ha en egen `stop()`-funksjon siden en `Spec` håndterer livssyklusen for en container ut av boksen. Ref: https://kotest.io/docs/extensions/test_containers.html#lifecycle